lib/logstash-logger/formatter/base.rb

```diff
@@ -25,7 +25,9 @@ def build_event(message, severity, time)
                   when LogStash::Event
                     data.clone
                   when Hash
-                    event_data = data.merge("@timestamp".freeze => time)
+                    event_data = data.clone
+                    event_data['message'.freeze] = event_data.delete(:message) if event_data.key?(:message)
```

Comment:
but for ruby 2.0.0p247 (2013-06-27 revision 41674)

```ruby
irb(main):001:0> "asd".object_id
=> 70313257930780
irb(main):002:0> "asd".object_id
=> 70313241344880
irb(main):003:0> "asd".freeze.object_id
=> 70313241338520
irb(main):004:0> "asd".freeze.object_id
=> 70313241348500
irb(main):005:0> "asd".freeze.object_id
```

And taking into account that library claims to support ruby 2.0 constanization still makes sense